docs/user/access-control.md

```diff
@@ -60,10 +73,28 @@ available in most Kubernetes distributions, you can find more information about
 that role
 [here](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#user-facing-roles).
 
+Apart from that role we need to create an additional one to have read access to the app repositories.
```

Comment:
I'd put the creation of the repositories-read role just after this sentence since you set the stage.

Reply:
> Apart from that role we need to create an additional one to have read access to the app repositories.

Additionally, we need to create a role to give read access to App Repositories and bind it to our service account.